packages/babylon/src/util/charCodes.js

```diff
@@ -0,0 +1,26 @@
+// @flow
```

Comment:
Can we make this a separate npm package.
I didn't find any module which does the inverse operation (char code -> char).

Reply:
I didn't find any either. We could technically publish it on NPM.

I don't want to use an existing module because we don't use every char code and I wanted to add functions for ranges. That way we only have the necessary.

Reply:
i would love to have this published separately and supporting all (most?) char codes

im always adding such constants in my projects, would be cool to just have a shareable module

> That way we only have the necessary.

imho keeping more is a non-issue, maintenance costs of such package should be really low too as this stuff never changes

Reply:
Ok, we can do this. Do we publish a module from the Babel org? It seems not really related.

Reply:
Depends on what kind of control do you want to have over it. Im really fine with any "owner" for it - if you feel it doesn't belong to babel maybe just release it under ur name, make it a dep and invite whole babel as collab.

Reply:
I created https://github.com/xtuc/charcodes, ping me if you need collab/publish.

Can I invite a whole Babel group at once?